README

```diff
@@ -11,14 +11,14 @@ This plugin is known to work with the following flavours of Vim:
   * Linux (tested on Ubuntu 12.04/12.10):
       * vim/gvim (from vim-gnome package version 7.3)
 
-  * Mac OS X (tested on Mountain Lion)
-      * Vim command-line (7.3 from Xcode)
+  * macOS (tested on Monterey)
+      * Vim command-line (8.2 from homebrew)
```

Comment:
This isn't really true because of https://github.com/lldb-tools/vim-lldb/issues/2

I was able to get this working with neovim pointing at system python on macOS, but that isn't really up to this plugin